packages/gatsby/src/redux/reducers/index.js

```diff
@@ -29,7 +30,7 @@ module.exports = {
   pages: pagesReducer,
   status: statusReducer,
   componentDataDependencies: require(`./component-data-dependencies`),
-  components: require(`./components`),
+  components: components,
```

Comment:
I'm maybe a bit picky, but would be worth imo making it consistent with all other reducers to be:

```suggestion
  components: componentsReducer,
```

(and therefore to `import { componentsReducer }` which mean also doing `export const componentsReducer` in the reducer file :)

Reply:
Good idea, changed :+1: